crates/forge/tests/cli/script.rs

```diff
@@ -1923,6 +1923,10 @@ contract SimpleScript is Script {
 
 // Asserts that the script runs with expected non-output using `--quiet` flag
 forgetest_async!(adheres_to_json_flag, |prj, cmd| {
+    if cfg!(feature = "isolate-by-default") {
+        return;
+    }
```

Comment:
Makes sense, I was looking for a way to have the fields of the JSON blob be more lenient yet still checked in Snapbox but couldn't get it to work.